perf/security/sds-tests/egress-sds/setup_nginx.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+function setup_nginx () {
+    local id="${1:?"please specify the gateway id"}"
+    local ns="${2:?"please specify the namespace"}"
+    local cs="${3:?"please specify the cluster"}"
```

Comment:
`ns` can be the short of namespace, well, `cluster` is short enough, just using `cluster` here, instead of using hard to interprete `cs`?

Reply:
Honestly this is not by choice. I kept it as `cs` just to be consistent with what we have in ingress-sds perf tests

Reply:
Yes, we use cs in other security tests as well. How about we keep it consistent here? We can rename variables for all the security tests in a followup PR.

Reply:
yes, sure!